src/corner_kick/public/main.js

```diff
@@ -0,0 +1,80 @@
+/**
+ * This file defines our Electron main process. This process
+ * is responsible for starting our renderer process.
+ *
+ * To learn more about the Electron process system, visit this link:
+ * https://electronjs.org/docs/tutorial/application-architecture
+ */
+
+const electron = require('electron');
+const app = electron.app;
+const BrowserWindow = electron.BrowserWindow;
+
+const path = require('path');
+const url = require('url');
+const isDev = require('electron-is-dev');
+
+const storage = require('electron-json-storage');
+
+let mainWindow;
+
+function main() {
+  // Needed to enable WebGL for all systems.
+  app.commandLine.appendSwitch('--ignore-gpu-blacklist');
```

Comment:
> Why are ignoring the GPU blacklist?

In order to take advantage of the WebGL context, which improves performance.

A comment was added to make this more clear:
```
// Needed to enable WebGL for all systems. We need WebGL to avoid
// using the CPU for visualizer features.
```